library/ssl_ticket.c

```diff
@@ -215,14 +215,14 @@ static int ssl_load_session( mbedtls_ssl_session *session,
     size_t cert_len;
 #endif /* MBEDTLS_X509_CRT_PARSE_C */
 
-    if( p + sizeof( mbedtls_ssl_session ) > end )
+    if( sizeof( mbedtls_ssl_session ) > (size_t)( end - p ) )
```

Comment:
Style: we normally put a space between `(type)` and `( value )`.

Reply:
I don't think that's the case: `egrep '(int|char|size_t)\)\(' library/*.c | wc -l` prints 307 while ` egrep '(int|char|size_t)\) \(' library/*.c | wc -l` only prints 93. Admittedly, one of them should print 0 if we were fully consistent, but most of the time we're not putting a space here.

Reply:
Oh. So it's `(foo_t) single_word` with a space (~90% prevalence) but `(foo_t)( more complex expression )` (~80% prevalence). Can we switch to a standard whitespace style in 3.0?